ShadowWizardMoneyGame/Mob.tscn

```diff
@@ -44,6 +44,7 @@ __meta__ = {
 scale = Vector2( 0.75, 0.75 )
 frames = SubResource( 1 )
 animation = "walk"
+frame = 1
```

Comment:
idk why but this comes up after running game?

Reply:
what specifically? I accidentally deleted your signal connection for when something enters mob body from a merge conflict in my previous PR that was merged to main. This is like a revert